Makefile

```diff
@@ -56,45 +56,44 @@ all-modules:
 
 .PHONY: gomoddownload
 gomoddownload:
-	@$(MAKE) for-all CMD="$(GOCMD) mod download"
+	@$(MAKE) for-all-target TARGET="moddownload"
 
 .PHONY: gotest
 gotest:
-	@$(MAKE) for-all CMD="make test test-unstable"
+	@$(MAKE) for-all-target TARGET="test test-unstable"
 
 .PHONY: gobenchmark
 gobenchmark:
-	@$(MAKE) for-all CMD="make benchmark"
+	@$(MAKE) for-all-target TARGET="benchmark"
 
 .PHONY: gotest-with-cover
 gotest-with-cover:
-	@$(MAKE) for-all CMD="make test-with-cover"
+	@$(MAKE) for-all-target TARGET="test-with-cover"
 	$(GOCOVMERGE) $$(find . -name coverage.out) > coverage.txt
 
 .PHONY: goporto
 goporto:
-	@$(MAKE) for-all CMD="make porto"
+	porto -w --include-internal ./
```

Comment:
nit: the target should be renamed to `porto` since it doesn't call go

Reply:
Haha :) I thought that we made a rule that if it is in the main `Makefile` we prefix with `go` and in the `Makefile.common` without :)